cmd/go-filecoin/client.go

```diff
@@ -190,8 +189,7 @@ be 2, 1 hour would be 120, and 1 day would be 2880.
 			abi.ChainEpoch(end),
 			price,
 			collateral,
-			// proof version (not circuit or size) should be all that is important here
-			constants.DevRegisteredWindowPoStProof,
+			status.MinerInfo.SealProofType,
```

Comment:
Looks good.  One thing to check is that you are now passing a seal proof instead of a post proof.  ProposeStorageDeal probably handles this internally but could be a problem.

Reply:
It actually should be a piece commitment proof type (which doesn't exist). Hopefully it's at least less confusing than it was.